src/pages/Embark/Landing.tsx

```diff
@@ -150,63 +150,74 @@ export const Landing: React.FC<{ language?: string }> = ({ language }) => {
         <Wrapper>
           <Container>
             {market === Market.Se && (
-              <>
-                <Card>
-                  <Headline>{textKeys.STARTPAGE_UNINSURED_HEADLINE()}</Headline>
-                  <Paragraph>{textKeys.STARTPAGE_UNINSURED_BODY()}</Paragraph>
-                  <ProceedButton
-                    size="lg"
-                    to={(language ? '/' + language : '') + '/new-member/new'}
-                  >
-                    {textKeys.STARTPAGE_UNINSURED_BUTTON()}
-                  </ProceedButton>
-                </Card>
-                <Card>
-                  <Headline>{textKeys.STARTPAGE_INSURED_HEADLINE()}</Headline>
-                  <Paragraph>{textKeys.STARTPAGE_INSURED_BODY()}</Paragraph>
-                  <ProceedButton
-                    size="lg"
-                    to={(language ? '/' + language : '') + '/new-member/switch'}
-                  >
-                    {textKeys.STARTPAGE_INSURED_BUTTON()}
-                  </ProceedButton>
-                </Card>
-              </>
+              <LandingPageCardsSe textKeys={textKeys} language={language} />
             )}
             {market === Market.No && (
-              <>
-                <Card>
-                  <Headline>
-                    {textKeys.STARTPAGE_HOME_CONTENT_HEADLINE()}
-                  </Headline>
-                  <Paragraph>
-                    {textKeys.STARTPAGE_HOME_CONTENT_BODY()}
-                  </Paragraph>
-                  <ProceedButton
-                    size="lg"
-                    to={
-                      (language ? '/' + language : '') +
-                      '/new-member/home-content'
-                    }
-                  >
-                    {textKeys.STARTPAGE_HOME_CONTENT_BUTTON()}
-                  </ProceedButton>
-                </Card>
-                <Card>
-                  <Headline>{textKeys.STARTPAGE_TRAVEL_HEADLINE()}</Headline>
-                  <Paragraph>{textKeys.STARTPAGE_TRAVEL_BODY()}</Paragraph>
-                  <ProceedButton
-                    size="lg"
-                    to={(language ? '/' + language : '') + '/new-member/travel'}
-                  >
-                    {textKeys.STARTPAGE_TRAVEL_BUTTON()}
-                  </ProceedButton>
-                </Card>
-              </>
+              <LandingPageCardsNo textKeys={textKeys} language={language} />
             )}
           </Container>
         </Wrapper>
       </LandingPageContainer>
     </Page>
   )
 }
+
+const LandingPageCardsSe: React.FC<{
+  textKeys: Record<string, any>
```

Comment:
I think we should use `TextKeyMap` from https://github.com/HedvigInsurance/web-onboarding/blob/master/src/utils/hooks/useTextKeys.ts - or just use `useTextKeys` inside these components instead.